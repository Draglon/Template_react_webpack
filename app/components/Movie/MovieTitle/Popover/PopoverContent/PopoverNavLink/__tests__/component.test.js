import React from 'react';

import PopoverNavLinkComponent from '../component';

describe('<PopoverNavLinkComponent />', () => {
  const props = {
    list: {
      name: 'list 1',
    },
    onClick: () => {},
  };
  const component = shallow(<PopoverNavLinkComponent {...props} />);

  it('should match snapshot', () => {
    expect(component).toMatchSnapshot();
  });
});
