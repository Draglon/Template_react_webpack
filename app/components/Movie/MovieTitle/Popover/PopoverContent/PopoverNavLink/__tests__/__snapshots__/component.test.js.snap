// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PopoverNavLinkComponent /> should match snapshot 1`] = `
<Button
  block={false}
  ghost={false}
  htmlType="button"
  loading={false}
  onClick={[Function]}
  type="link"
>
  list 1
</Button>
`;
